mlr-org/_posts/2020-08-26-introducing-mlr3cluster-cluster-analysis-package/introducing-mlr3cluster-cluster-analysis-package.Rmd

```diff
@@ -214,9 +214,9 @@ learner$train(task)
 autoplot(learner)
 
 # More advanced options from `factoextra::fviz_dend`
-autoplot(learner,
-  k = learner$param_set$values$k, rect_fill = TRUE,
-  rect = TRUE)
+autoplot(learner, k = learner$param_set$values$k,
+  rect=TRUE, rect_border=c("blue")
+)
```

Comment:
Note that you can simply use our {styler.mlr} package as [shown here](https://github.com/mlr-org/mlr3/wiki/Style-Guide).